### PR TITLE
media-video/motion: version bump to 4.1.1

### PR DESCRIPTION
**Pull request by vilhelmgray**
Closes: https://bugs.gentoo.org/639526

This version adds support for the sqlite and webp USE flags.

### CODE REVIEW

File: media-video/motion/motion-4.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		$(use_with sqlite sqlite3) \
+		$(use_with v4l v4l2) \
+		$(use_with webp) \
+		--without-optimizecpu
+}
+
+src_install() {
+	emake \
+		DESTDIR="${D}" \
+		docdir=/usr/share/doc/${PF} \
```
Comment (mgorny) [inline media-video/motion/motion-4.1.1.ebuild]: Why do you need to redefine this? Doesn't configure handle it?

File: media-video/motion/motion-4.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		examplesdir=/usr/share/doc/${PF}/examples \
+		install
+
+	newinitd "${FILESDIR}"/${PN}.initd-r3 ${PN}
+	newconfd "${FILESDIR}"/${PN}.confd-r1 ${PN}
+
+	systemd_dounit "${FILESDIR}"/${PN}.service
+	systemd_dounit "${FILESDIR}"/${PN}_at.service
```
Comment (mgorny) [inline media-video/motion/motion-4.1.1.ebuild]: That installs the unit with the wrong name.

File: media-video/motion/motion-4.1.1.ebuild (hunk 1 above)
Comment (vilhelmgray) [reply media-video/motion/motion-4.1.1.ebuild]: The configure defaults to the doc directory to `/usr/share/doc/motion/` which can get confusing with multiple motion packages. The line `docdir=/usr/share/doc/${PF}` is used to make sure the doc directory is not missing the motion version.

Comment (mgorny) [reply media-video/motion/motion-4.1.1.ebuild]: Ok, I see that upstream has custom `Makefile.in` that doesn't use `@docdir@` correctly.

**Pull request closed without merging**